docs/main/concepts/data-engines/local-engine.md

```diff
@@ -85,17 +85,17 @@ A quick summary of the steps to restore include:
    velero restore create rbb-01 --from-backup bbb-01 -l app=test-velero-backup
    ```
 
-## Limitations (or Roadmap items) of OpenEBS Local Engine
+## Limitations (or Roadmap items) of OpenEBS Local Storage
```

Comment:
This section too needs updates. Review with Abhinandan/Niladri and address in separate PR.

Reply:
@niladrih @Abhinandan-Purkait please review and advise the changes to be made. Thanks.